app/controllers/application_controller.rb

```diff
@@ -19,7 +20,34 @@ def get_all_facilities
       #This can be written as follows:
       # session[:login_status] = 'not logged in' unless session[:login_status] == 'logged in'
       # @allfacilities = Facility.all
-
+    
   	  
   end
+  def get_provided_services
+     @surgery = 0
+     @fp = 0
+     @maternity = 0
+     @dental = 0
+     @other = 0
+     @allservices = 0
+     services_provided = FacilityService.all
```

Comment:
try returning:

`{ maternity: FacilitiyService.all.count { |f| f.service_id == 1 }, dental: FacilityService.all.count { |f| f.service_id == 2 }`

and so on...